test/serializer/optimized-functions/LoopBailout19,js

```diff
@@ -0,0 +1,17 @@
+function fn(x, oldItems) {
+  var items = [];
+  for (var i; ; ) {
+    i++;
+    if (i > x) {
+      breakl
```

Comment:
Is this intentional? If so, a comment will help the reader.

Reply:
No it was not.

Reply:
Fixed the test, I didn't save the file before committing.